modules/ROOT/pages/node-governance.adoc

```diff
@@ -0,0 +1,27 @@
+= Node Governance
+
+Node governance surfaces the security and compliance posture of each agent, broker, and Model Context Protocol (MCP) server in the agent network. View Flex Gateway protection and applied policies and follow links to API Manager to view policies.
```

Comment:
I wonder if MCP is common enough (now) to only define once early on.

Reply:
I don't think people just start in the Overview and then read in order. I was treating every page like the first page.